src/utils/pureFunctions.ts

```diff
@@ -11,6 +11,17 @@ export const shortenString = (
   return start + separator + end;
 };
 
+export const truncateString = (
```

Comment:
truncated string will not have the intended length because the '...'.length isn't taken into account.

Reply:
So can we add `-3`?